ui/apps/pixano/src/routes/[dataset]/dataset/[itemId]/+page.svelte

```diff
@@ -79,14 +81,19 @@ License: CECILL-C
                   video.sort((a, b) => a.data.frame_index - b.data.frame_index);
                 } else {
                   item_type = "vqa";
+                  //TMP
```

Comment:
this is a workaround because refcoco is currently wrongly built with "list[Image]" instead of plain "Image"
this should not be kept IMO